tests/baselines/reference/ambientStatement1.errors.txt

```diff
@@ -9,6 +9,6 @@ tests/cases/compiler/ambientStatement1.ts(4,20): error TS1039: Initializers are
 !!! error TS1036: Statements are not allowed in ambient contexts.
         
         	export var v1 = () => false;
-        	              ~

```

Comment:
How is this readonly declaration?

Reply:
It's not, the error span location was just normalized for the initializer forbidden error to always be on the initializer instead of sometimes on the initializer and sometimes on the equals token.

Reply:
@weswigham makes sense. Sorry somehow I perceived as error removed. My bad.